site/content/integrate/apps/quickstart/quick-start-serverless/_index.md

```diff
@@ -1,436 +1,93 @@
 
-`golang-middleware` template makes use of adding boilerplate handlers to
-`http.DefaultServeMux`, so we just need to add our handlers and serve.
+## Use the app
 
-```go
-var deployType apps.DeployType
+You can now use the `/hello-serverless send` command with an optional `message` argument. This will cause the app's bot user to DM you a response:
 
-func InitApp(dt apps.DeployType) {
-	// Returns the Channel Header and Command bindings for the App.
-	http.HandleFunc("/bindings", httputils.HandleJSONData(bindingsData))
+![image](response.png)
 
-	// The form for sending a Hello message.
-	http.HandleFunc("/send/form", httputils.HandleJSONData(formData))
+If you included `message` in the slash command, that would also be included in the response.
 
-	// The main handler for sending a Hello message.
-	http.HandleFunc("/send/submit", send)
+## Review the app
 
-	deployType = dt
-}
+To understand the app, examine the following elements:
 
-func send(w http.ResponseWriter, req *http.Request) {
-	creq := apps.CallRequest{}
-	json.NewDecoder(req.Body).Decode(&creq)
+### Manifest
 
-	message := fmt.Sprintf("Hello from a serververless app running as %s!", deployType)
-	v, ok := creq.Values["message"]
-	if ok && v != nil {
-		message += fmt.Sprintf(" ...and %s!", v)
-	}
+The app must provide a manifest, which declares app metadata. In this example, the following permissions are requested via the `/manifest.json` endpoint:
 
-	asBot := appclient.AsBot(creq.Context)
-	asBot.DM(creq.Context.ActingUserID, message)
-
-	httputils.WriteJSON(w,
-		apps.NewOKResponse(nil, "Created a post in your DM channel."))
-}
-```
-
-#### `go.mod` and `go.sum`
-
-Since package `function` is built independently by OpenFaaS, add its own `go.mod` for its ddependencies:
-
-```
-module {{path-to-your-module}}/function
-
-go 1.16
-
-require github.com/mattermost/mattermost-plugin-apps v0.8.0
-```
-
-### 4. `./aws`
-#### `main.go`
-```go
-package main
-
-import (
-    "net/http"
-
-    "github.com/aws/aws-lambda-go/lambda"
-    "github.com/awslabs/aws-lambda-go-api-proxy/httpadapter"
-    "github.com/mattermost/mattermost-plugin-apps/apps"
-
-    "{{path-to-your-module}}/function"
-)
+- Create posts as a bot.
+- Create slash commands.
 
-func main() {
-    function.InitApp(apps.DeployAWSLambda)
+### Bindings and locations
 
-    // run with the Lambda request re-marshaled back into an http.Request so that it works with mux.
-    lambda.Start(httpadapter.New(http.DefaultServeMux).Proxy)
-}
-```
+Locations are named elements in the Mattermost user interface. Bindings specify how an app's calls should be displayed and invoked from these locations. This app uses only uses the slash command location.
 
-#### `go.mod`
-```
-module {{path-to-your-module}}/aws
+### Functions
 
-go 1.16
+Functions handle user events and webhooks. The webhooks app uses only one main function via a slash command:
```

Comment:
Same concern here about webhooks vs. serverless. It's confusing to see a reference to webhook Apps in a doc about serverless Apps.

Reply:
Fixed this.